x-pack/plugins/security_solution/public/timelines/components/flyout/index.test.tsx

```diff
@@ -51,54 +42,32 @@ describe('Flyout', () => {
 
   describe('rendering', () => {
     test('it renders correctly against snapshot', () => {
-      const wrapper = shallow(
+      const { asFragment } = render(
         <TestProviders>
           <Flyout {...props} />
         </TestProviders>
       );
-      expect(wrapper.find('Flyout')).toMatchSnapshot();
+      expect(asFragment()).toMatchSnapshot();
     });
 
     test('it renders the default flyout state as a bottom bar', () => {
-      const wrapper = mount(
+      render(
         <TestProviders>
           <Flyout {...props} />
         </TestProviders>
       );
 
-      expect(wrapper.find('[data-test-subj="flyoutBottomBar"]').first().text()).toContain(
-        'Untitled timeline'
-      );
-    });
-
-    test('it does NOT render the fly out bottom bar when its state is set to flyout is true', () => {
```

Comment:
I could not find `data-test-subj="flyout-button-not-ready-to-drop"` in any component, so I assumed this test has become obsolete now.

Reply:
ya I have no idea what this refers to, probably has been missing for a while